public/app/core/components/AppChrome/News/NewsContainer.tsx

```diff
@@ -41,7 +41,7 @@ export function NewsContainer({ className }: NewsContainerProps) {
                   variant="secondary"
                   fill="text"
                   onClick={onToggleShowNewsDrawer}
-                  aria-label={selectors.components.Drawer.General.close}
+                  data-testid={selectors.components.Drawer.General.close}
                 />
```

Comment:
(sorry can't select where to actually leave the comment 😬 )
This `Button` component can be replaced by `IconButton` (that's one of the recommendations on this [migration post](https://github.com/grafana/grafana/issues/36523#issuecomment-1819160512)) and it should have a tooltip so users with screen readers still know what it is for.